changelogs/fragments/5733-rax-deprecation-notice.yml

```
@@ -0,0 +1,27 @@
+deprecated_features:
+  - rax_cbs_attachments - module relies on deprecates library ``pyrax`` and will be removed in community.general 9.0.0 (https://github.com/ansible-collections/community.general/pull/5733).
```

```suggestion
  - rax_cbs_attachments - module relies on deprecates library ``pyrax``. Unless maintainers step up to work on the module, it will be marked as deprecated in community.general 7.0.0 and removed in version 9.0.0 (https://github.com/ansible-collections/community.general/pull/5733).
```
Same for the other lines.